### PR TITLE
Add recipe for Lingua

### PR DESCRIPTION
**Pull request by Vrondir**

### CODE REVIEW

**Comment by aarroyoc** [comment]
LGTM. Comes from: https://codein.withgoogle.com/dashboard/task-instances/5977961652551680/

File: haiku-apps/lingua/lingua-2.0.0.recipe [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+COPYRIGHT="The Aevum Network authors"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="246414192994f9ef8f7d1afcaa705b5e80045c7b"
+SOURCE_URI="https://github.com/HaikuArchives/Lingua/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="1c1b68d219c44b0f3df33246169211057dd67d62d2b5117c2effcbb984161c5a"
+SOURCE_FILENAME="Lingua-$srcGitRev.tar.gz"
+SOURCE_DIR="Lingua-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 x86 ?x86_64"
```
Comment (owenca) [inline haiku-apps/lingua/lingua-2.0.0.recipe]: `ARCHITECTURES="x86_gcc2 x86 !x86_64"`

Comment (Begasus) [inline haiku-apps/lingua/lingua-2.0.0.recipe]: @owenca thanks for the pointer!
```diff
-ARCHITECTURES="x86_gcc2 x86 ?x86_64"
+ARCHITECTURES="x86_gcc2 x86 !x86_64"
```

**Comment by extrowerk** [comment]
```
waiting for build package lingua-2.0.0-1 to be activated
waiting for build package lingua_debuginfo-2.0.0-1 to be activated
Building ...
mwcc -c AboutWindow.cpp -I./ -i-  -O3 -w off -o obj.ppc/AboutWindow.o
mwcc -c DeskbarView.cpp -I./ -i-  -O3 -w off -o obj.ppc/DeskbarView.o
make: mwcc: Command not found
Makefile:70: recipe for target 'obj.ppc/AboutWindow.o' failed
make: *** [obj.ppc/AboutWindow.o] Error 127
make: *** Waiting for unfinished jobs....
make: mwcc: Command not found
Makefile:70: recipe for target 'obj.ppc/DeskbarView.o' failed
make: *** [obj.ppc/DeskbarView.o] Error 127
Warning: Command '['bash', '-c', '. /wrapper-script']' returned non-zero exit status 2
Error: Build has failed - stopping.
```

**Comment by extrowerk** [comment]
Lingua is broken on 64 bit, please either fix it or disable the recipe for 64 bit.

**Comment by extrowerk** [comment]
64 bit doesnt defines BEPC, so probably the makefile needs some changes for 64 bit.

File: haiku-apps/lingua/lingua-2.0.0.recipe [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+COPYRIGHT="The Aevum Network authors"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="246414192994f9ef8f7d1afcaa705b5e80045c7b"
+SOURCE_URI="https://github.com/HaikuArchives/Lingua/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="1c1b68d219c44b0f3df33246169211057dd67d62d2b5117c2effcbb984161c5a"
+SOURCE_FILENAME="Lingua-$srcGitRev.tar.gz"
+SOURCE_DIR="Lingua-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 x86 !x86_64"
```
Comment (Begasus) [inline haiku-apps/lingua/lingua-2.0.0.recipe]: If you want to keep x86 as primary architecture here mark it as broken (no way to test it) or remove it from the line:
```diff
-ARCHITECTURES="x86_gcc2 x86 !x86_64"
+ARCHITECTURES="x86_gcc2 !x86 !x86_64"
```

File: haiku-apps/lingua/lingua-2.0.0.recipe [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+SOURCE_DIR="Lingua-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 !x86 !x86_64"
+
+PROVIDES="
+	lingua = $portVersion
+	app:Lingua = $portVersion
+	"
+REQUIRES="
+	haiku >= $haikuVersion
```
Comment (korli) [inline haiku-apps/lingua/lingua-2.0.0.recipe]: @Begasus $haikuVersion is not needed here.

Comment (Begasus) [reply haiku-apps/lingua/lingua-2.0.0.recipe]: Eeps, didn't see that one, will fix :)

**Pull request merged**